docs/installation_configuration_guide/model/Lite.md

```diff
@@ -14,7 +14,7 @@ sidebar_label: "Lite"
 | pollDuration             | Low        | duration | 100ms                                         | [Poll duration](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#poll(java.time.Duration)) of Kafka consumer | 
 | shutdownTimeout          | Low        | duration | 10s                                           | How long to wait for graceful shutdown                                                                                                               |
 | interpreterTimeout       | Low        | duration | 10s                                           | Timeout of invocation of scenario (including enrichers) for events consumed in one poll                                                              |
-| publishTimeout           | Low        | duration | 10s                                           | Timeout on producing resulting event to Kafka                                                                                                        |
+| publishTimeout           | Low        | duration | 5s                                            | Timeout on producing resulting event to Kafka                                                                                                        |
```

Comment:
Yo didn't mention about change default behavior?

Reply:
I should've added a comment for the reviewer, sorry for the confusion. I didn't change the behaviour, just was looking at `KafkaInterpreterConfig` and saw that the publishTimeout default arg didn't match with the docs. I didn't see an overwrite, so I assumed the docs are incorrect.